com.eviware.soapui.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: com.eviware.soapui
+
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
```

Comment:
this runtime has been EOL for over a year.

Reply:
can't is run with 17?

Reply:
It was reporting problems in the console with OpenJFX, but maybe is an issue with the JFX from `org.freedesktop.Sdk.Extension.openjdk17`, but it seams to launch correctly.

Since in their [README](https://github.com/SmartBear/soapui/blob/next/README.md#prerequisites) they report to require JRE 16 I haven't investigated it any further.
Another option I had in mind was to use their installer that includes a full version of Java JRE, but I don't think it should be necessary.

But I agree we can leave it with openjdk17 until somebody reports an actual issue with the java version.